app/models/edm/web_resource.rb

```diff
@@ -43,6 +43,7 @@ class WebResource
     validates :media, presence: true, if: :published?
     validates :edm_rights, presence: true, unless: :media_blank?
     validate :europeana_supported_media_mime_type, unless: :media_blank?
+    validate :media_size_permitted, unless: :media_blank?
```

Comment:
Let's run this validation before the mime type one, to prevent making the `file` system call if the file's too large.